common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -936,12 +936,14 @@ static jlong NativeCrypto_EVP_PKEY_new_RSA(JNIEnv* env, jclass, jbyteArray n, jb
         return 0;
     }
 
+#if defined(CONSCRYPT_JNI_TRACE)
     if (conscrypt::trace::kWithJniTrace) {
         if (p != nullptr && q != nullptr) {
             int check = RSA_check_key(rsa.get());
```

Comment:
At some point in my infinite spare time, I'll probably introduce new `RSA_new_foo` functions and try to get you all to switch to them, which will internally call `RSA_check_key` and refuse to construct invalid ones. `RSA` objects that fail our internal invariants act really weird and usually fail every operation. :-)

But that's tangential to this PR.